html/pfappserver/lib/pfappserver/Base/Form/Role/SourceLocalAccount.pm

```diff
@@ -40,8 +40,35 @@ has_field 'local_account_logins' => (
     },
 );
 
+has_field 'hash_passwords' => (
+    type => 'Select',
+    label => 'Database passwords hashing method',
+    options =>
+    [
+     { value => 'plaintext', label => 'Plaintext' },
+     { value => 'ntlm', label => 'NTLM' },
+     { value => 'bcrypt', label => 'Bcrypt' },
+    ],
+    required => 1,
+    element_class => ['input-small'],
+    default => 'ntlm',
```

Comment:
`hash_passwords=bcrypt` in `pf.conf.defaults`, we need to do same thing for sources.

Reply:
Even better idea, why don't we take the value currently configure in `pf::ConfigStore::Pf->new->read("advanced")->{hash_passwords}`

Reply:
Actually, lets scratch my previous comment.

This should point to the value in the source model metadata like the other sources